config/prod/elevatems_features.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v8.j2
+stack_name: elevatems-features
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "ElevateMS"
+  # The resource owner
+  OwnerEmail: "meghasyam@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "{{stack_group_config.default_vpc}}"
+  KeyName: "{{stack_group_config.default_key_pair}}"
+  # AMIId: "{{stack_group_config.default_ami}}"
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/pricing/on-demand/)
+  InstanceType: "c4.8xlarge"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "240"
+  # (Optional) Base instance on AMI (default: ami-082278746f893d99c) (supported distros: AWS linux)
+  AMIId: "ami-01a01f46103fb6737"
```

Comment:
is this a new AMI from your personal account? has it been shared to the scicomp account? is the root volume encrypted?

Reply:
This is not a personal AMI. I just linked the tag for the latest Ubuntu AMI (US West Oregon) from http://www.louisaslett.com/RStudio_AMI/

I am unsure of the specifications i.e, root volume encryption

Reply:
`us-east-1` would be better, which is  `ami-0226a8af83fcecb43`